.vib/phpmyadmin/cypress/cypress/integration/phpmyadmin_spec.js

```diff
@@ -0,0 +1,72 @@
+/// <reference types="cypress" />
+import {
+    random
+} from './utils'
+
+it('allows the user to log out', () => {
+    cy.login();
+    cy.contains('[role="alert"]', 'Access denied').should('not.exist');
+    cy.get('a[title="Log out"]').should('be.visible').click();
+    cy.get('#login_form').should('be.visible');
+})
+
+it('allows creating a database and a table', () => {
+    cy.login();
+    cy.visit('index.php?route=/server/databases');
+    cy.fixture('testdata').then((td) => {
+        cy.get('#text_create_db').type(`${td.databaseName}.${random}`);
+        cy.get('#buttonGo').click();
+        cy.get('.lock-page [type="text"]').type(`${td.tableName}.${random}`);
+        cy.get('.lock-page [type="number"]').clear().type(td.columnNumber);
+        cy.contains('[type="submit"]', 'Go').click();
+        cy.get('#field_0_1').type(`${td.columnName}.${random}`);
+        cy.get('.btn-primary').click();
+        cy.visit('index.php');
+        cy.contains('a', `${td.databaseName}.${random}`).scrollIntoView().click({
+            force: true
+        });
+        cy.contains('#pma_navigation_tree_content', `${td.tableName}.${random}`).should('be.visible').click({
+            force: true
+        })
+        cy.contains('.table-responsive-md', `${td.columnName}.${random}`).should('be.visible');
+    });
+})
+
+it('allows importing a table and executing a query', () => {
+    cy.login();
+    cy.visit('index.php?route=/database/structure&server=1&db=mysql')
+    cy.contains('Import').click();
+    cy.get('#input_import_file').should('be.visible').selectFile('cypress/fixtures/testdata.sql', {
+        force: true
+    });
+    cy.get('#buttonGo').click();
+    cy.contains('No database selected', '[role="alert"]').should('not.exist');
+    cy.contains('Import has been successfully finished');
+    cy.fixture('testdata').then((td) => {
+        cy.contains('[title="Browse"]', 'authors').should('be.visible').click();
```

Comment:
You are loading a fixture but actually not using it. I guess that you wanted to include an entry for the Database name, `authors`, right?